### PR TITLE
Nav colour change

### PR DESCRIPTION
**Pull request by zeftilldeath**
## What does this change?
No more bluey grey in the nav. Reduces confusion this extra family of greys adds. I've replaced all instances of them on frontend with equivalent neutrals. Aware the nav tones are in some other places, so they need to be replaced.

### Todos
- [x] identity-frontend @walaura
- [ ] manage.gu @AWare 
- [ ] 404 page Me
- [ ] Default twitter card Me
- [ ] UX decks Gustavo

### Screenshots

screen shot 2018-09-13 at 15 24 54

screen shot 2018-09-12 at 16 15 20

screen shot 2018-09-12 at 16 15 33

### CODE REVIEW

**Review by sammorrisdesign-zz — APPROVED** [review]
YES DARLING

**Comment by walaura** [comment]
I've dropped in a checklist for tomorrow!

File: static/src/stylesheets/layout/_side-margins.scss [hunk 1]
```diff
@@ -30,7 +30,7 @@
             content: '';
             position: absolute;
             z-index: 1;
-            background: rgba($brightness-7,  .05);
+            background: rgba(0, 0, 0,  .03);
```
Comment (walaura) [inline static/src/stylesheets/layout/_side-margins.scss]: red_excited tears

**Comment by zeftilldeath** [comment]
Things that will need to follow suit:

- [ ] - 404 page
- [ ] - Default twitter card
- [ ] - Identity
- [ ] - Supporter Experience

File: static/src/stylesheets/layout/_side-margins.scss (hunk 1 above)
Comment (zeftilldeath) [reply static/src/stylesheets/layout/_side-margins.scss]: What?

Comment (walaura) [reply static/src/stylesheets/layout/_side-margins.scss]: i'm happy zef this is me being happy

Comment (zeftilldeath) [reply static/src/stylesheets/layout/_side-margins.scss]: It's so scary!

**Comment by AWare** [comment]
So it'll be 👌🏻 if I change these colours in manage-frontend and dotcom-rendering?

| old hex | old name | new name | 
|---------|----------|----------|
| #bbcdd3 | nav-dark | brightness-86 | 
| #dae4e7 | nav-bold | brightness-93 | 
| #e9eff1 | nav-faded| brightness-97 |

**Review by walaura — APPROVED** [review]
Can you take a screenshot of https://profile.thegulocal.com/consents ? Bit scared about how that one might look now

File: static/src/stylesheets/module/_adslot.scss [hunk 2]
```diff
@@ -58,6 +58,7 @@
 
 .top-banner-ad-container {
     background-color: $brightness-100;
+    border-bottom: 1px solid $brightness-86;
```
Comment (walaura) [inline static/src/stylesheets/module/_adslot.scss]:
```
°˖✧◝(⁰▿⁰)◜✧˖°
```

File: static/src/stylesheets/amp/nav/_menu-group.scss [hunk 3]
```diff
@@ -10,17 +10,17 @@
...
 .menu-group--membership,
 .menu-group--editions {
-    background-color: $nav-bold;
+    background-color: $brightness-93;
     padding-bottom: 0;
 }
 
 .menu-group--editions .menu-group {
-    background-color: darken($nav-faded, 10%);
+    background-color: darken($brightness-97, 10%);
```
[the remark this answers is not in the document]
Comment (zeftilldeath) [reply static/src/stylesheets/amp/nav/_menu-group.scss]: Not without inventing another grey unfortunately!

[the remark this answers is not in the document]
Comment (AWare) [reply static/src/stylesheets/amp/nav/_menu-group.scss]: Is `darken()` essential, as we don't currently have the capability on dotcom-rendering.

[the remark this answers is not in the document]
Comment (zeftilldeath) [reply static/src/stylesheets/amp/nav/_menu-group.scss]: brightness-86 should do the trick!

**Pull request merged**